FlowCrypt/Functionality/Services/Local Private Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -84,6 +84,9 @@ final class InMemoryPassPhraseProvider: InMemoryPassPhraseProviderType {
     }
 
     func save(passPhrase: PassPhrase) {
+        // First remove pass phrase if it's already saved.
+        // (This is to ensure that it updates pass phrase value correctly)
+        passPhrases.remove(passPhrase)
```

Comment:
What about using `passPhrases.update(with: passPhrase)` here?

Reply:
Yeah, that's clearer :) Thanks.